packages/transformers/postcss/src/loadConfig.js

```diff
@@ -0,0 +1,149 @@
+// @flow
+import type {Config, PluginOptions} from '@parcel/types';
+import type {PluginLogger} from '@parcel/logger';
+import path from 'path';
+
+import loadExternalPlugins from './loadPlugins';
+
+const MODULE_BY_NAME_RE = /\.module\./;
+
+async function configHydrator(
+  configFile: any,
+  config: Config,
+  options: PluginOptions,
+) {
+  // Use a basic, modules-only PostCSS config if the file opts in by a name
+  // like foo.module.css
+  if (configFile == null && config.searchPath.match(MODULE_BY_NAME_RE)) {
+    configFile = {
+      plugins: {
+        'postcss-modules': {},
+      },
+    };
+  }
+
+  if (configFile == null) {
+    return;
+  }
+
+  // Load the custom config...
+  let modules;
+  let configPlugins = configFile.plugins;
+  if (
+    configPlugins != null &&
+    typeof configPlugins === 'object' &&
+    configPlugins['postcss-modules'] != null
+  ) {
+    modules = configPlugins['postcss-modules'];
+    // $FlowFixMe
```

Comment:
I don't think this is necessary anymore, when I removed it I didn't get any Flow errors.